src/cmd/serve.rs

```diff
@@ -606,24 +620,24 @@ pub fn serve(
     })
     .expect("Error setting Ctrl-C handler");
 
-    use notify::DebouncedEvent::*;
-
-    let reload_sass = |site: &Site, path: &Path, partial_path: &Path| {
-        let msg = if path.is_dir() {
-            format!("-> Directory in `sass` folder changed {}", path.display())
-        } else {
-            format!("-> Sass file changed {}", path.display())
-        };
+    let reload_sass = |site: &Site, paths: &Vec<&PathBuf>| {
+        let combined_paths =
+            paths.iter().map(|p| p.display().to_string()).collect::<Vec<String>>().join(", ");
+        let msg = format!("-> Sass file(s) changed {}", combined_paths);
         console::info(&msg);
         rebuild_done_handling(
             &broadcaster,
             compile_sass(&site.base_path, &site.output_path),
-            &partial_path.to_string_lossy(),
+            &site.sass_path.to_string_lossy(),
```

Comment:
@Keats , a question on this change and a similar one in `reload_templates`. Previously we passed partial paths here, but now I'm passing an absolute path to the root of the sass directory. I couldn't find (or resolve, for some reason) protocol docs for livereload v7. Is this change okay? Functionally it seems to work.